Public/Src/IDE/Generator/MsbuildFile.cs

```diff
@@ -86,7 +90,7 @@ internal abstract class MsbuildFile
             // Add '.g' suffix only if the projects files will be in the spec root
             if (Context.CanWriteToSrc)
             {
-                projectExtension = ".g" + projectExtension;
+                projectExtension = ".2.g" + projectExtension;

```

Comment:
Is this to allow for old/new file co-existence?

Reply:
Indeed

Reply:
I'll change it back to `.g` once the old generator is deprecated